### PR TITLE
Jb48425 remove statefs

### PR DESCRIPTION
**Pull request by blammit**
@rainemak @monich review please? Are there any issues with changing privileges?

### CODE REVIEW

File: src/pages/components/ResourceController.qml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /****************************************************************************
 **
-** Copyright (C) 2013 Jolla Ltd.
+** Copyright (C) 2013-2019 Jolla Ltd.
```
Comment (rainemak) [inline src/pages/components/ResourceController.qml]: Could you lower-case c from (c).

We discussed once with @spiiroin that also years should be marked with spaces around the dashes. So, it should look like:

> ** Copyright (c) 2013 - 2019 Jolla Ltd.

:smile:

File: src/pages/components/TextSelectionController.qml [hunk 2]
```diff
@@ -210,14 +211,23 @@ MouseArea {
...
-    ContextProperty {
+
+    OfonoNetworkRegistration {
         id: cellular2Status
-        property bool disabled: value == "disabled" || value == undefined
-        key: "Cellular_1.Status"
+
+        property bool registered: status == "registered" || status == "roaming"
+
+        modemPath: ofonoManager.modems[1] || ""
```
Comment (rainemak) [inline src/pages/components/TextSelectionController.qml]: Didn't realized that indexing out of bounds doesn't cause issues. As out-of-bounds indexing returns undefined this should be good => at least no errors in the when undefined is assigned to the modem path.

**Comment by rainemak** [comment]
Minor remarks / notes. Please check those. Approving already.

:+1:  LGTM

**Comment by rainemak** [comment]
Still LGTM

**Comment by blammit** [comment]
Thanks @rainemak, could you merge and tag please?

**Pull request merged**